src/Chirp.Razor/Pages/Shared/_Layout.cshtml

```diff
@@ -1,4 +1,14 @@
-﻿<!DOCTYPE html>
+﻿@using System.Security.Principal
+@using Microsoft.AspNetCore.Authentication.OpenIdConnect
+@using Microsoft.Extensions.Options
+@using Microsoft.Identity.Web
+
+@inject IOptionsMonitor<MicrosoftIdentityOptions> AzureADB2COptions
+
+@{
+    var options = AzureADB2COptions.Get(OpenIdConnectDefaults.AuthenticationScheme);
+}
```

Comment:
Can you perphaps please explain what this does?

Reply:
Will look further in to it tomorrow at our meeting, got it from here https://techcommunity.microsoft.com/t5/azure-developer-community-blog/migrating-azure-ad-b2c-integration-going-from-net-core-3-1-to/ba-p/1983575